infrastructure/lib/atat-iam-stack.ts

```diff
@@ -0,0 +1,184 @@
+import * as iam from "@aws-cdk/aws-iam";
+import * as cdk from "@aws-cdk/core";
+import * as custom from "@aws-cdk/custom-resources";
+
+export class AtatIamStack extends cdk.Stack {
+  private readonly outputs: cdk.CfnOutput[] = [];
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+    this.templateOptions.description = "Creates the IAM permissions for ATAT developers, testers, and auditors.";
+
+    // job-function/ReadOnlyAccess is actually a somewhat-dangerous IAM policy that
+    // allows users to view sensitive data (such as Lambda environment variables or
+    // some Secrets Manager Secrets). ViewOnlyAccess is more restrictive.
+    // All roles will have ViewOnly access.
+    const awsManagedViewOnlyPolicy = iam.ManagedPolicy.fromAwsManagedPolicyName("job-function/ViewOnlyAccess");
+    const awsManagedLogsReadPolicy = iam.ManagedPolicy.fromAwsManagedPolicyName("CloudWatchLogsReadOnlyAccess");
+    const awsManagedAuditorPolicy = iam.ManagedPolicy.fromAwsManagedPolicyName("SecurityAudit");
+    const awsManagedAdminAccessPolicy = iam.ManagedPolicy.fromAwsManagedPolicyName("AdministratorAccess");
+    const awsManagedCfnFullAccessPolicy = iam.ManagedPolicy.fromAwsManagedPolicyName("AWSCloudFormationFullAccess");
+
+    // In the future, we may want to consider moving to a mechanism where the various
+    // resources, when created, grant access to these administrative roles rather than
+    // having us globally grant access at this level. For example, at this point, it
+    // is appropriate for our users to have access to read all tables directly but
+    // will it always? Might we eventually store sensitive data there?
+    // Eventually, many of these actions may require access to a KMS Key.
+    const generalReadAccess = new iam.ManagedPolicy(this, "GeneralReadAccess", {
+      statements: [
+        new iam.PolicyStatement({
+          sid: "DynamoDBItemAccess",
+          effect: iam.Effect.ALLOW,
+          actions: ["dynamodb:*GetItem", "dynamodb:PartiQLSelect", "dynamodb:Scan", "dynamodb:Query"],
+          resources: [`arn:${cdk.Aws.PARTITION}:dynamodb:*:${cdk.Aws.ACCOUNT_ID}:table/*`],
+        }),
+        new iam.PolicyStatement({
+          sid: "APIGatewayRestApiReadAccess",
+          effect: iam.Effect.ALLOW,
+          actions: ["apigateway:GET"],
+          resources: [`arn:${cdk.Aws.PARTITION}:apigateway:*::/restapis*`],
+        }),
+      ],
```

Comment:
`sqs:ListQueues` will be available through the `ViewOnlyPolicy`. Do we need to grant `sqs:*` or is there something more reasonable? Can we solve it in the SQS ticket?

Reply:
I guess the question is _who_ will need to perform those actions. Everyone, or developers? Perhaps these RW actions can go in the developer RW access policy? In any case, since SQS is not on the `develop` branch today and adding permissions for a service we're not currently using feels antithetical to the goals of trimming down IAM to what's required as part of this PR :) (ignore the CloudFormation role's permissions when you read that sentence). But I want to avoid speculatively granting access to things.

Reply:
I've asked @hannahposeyscholl-ccpo to create a future ticket referencing this conversation.